s32217-OskarNiedzwiedzki/zadanie3.txt

```diff
@@ -0,0 +1,86 @@
+<<<<<<< HEAD
+Rozgrzeweczka:
+- ls -a
+  -[Wypisało 16 plików/katalogów]
+
+- ls -a | grep l [Nie miałem plików na D]
+  -[Wypisało 7 plików mających literkę l w sobie]
+
+- ls -a | grep l > lista_plikow_z_lierka_l.txt
+  - Plik się utworzył, zawiera wszystkie pliki ktore zawierają w swojej nazwie literkę l
+  - Potok [|] przekazuje standardowy strumień wyjścia 
+  - > przekazuje standardowy strumień wyjścia do pliku do nadpisania
+
+- ls -a | grep l | tee lista_plikow_z_literka_l_2.txt
+ - plik się utworzył, nie zawiera on jedynie samego siebie
+ - tee czyta ze standardowego wejścia i zapisuje na standardowe wyjście oraz do pliku
+
+1.
+- ps - wypisuje aktualne procesy
+- ps -a -wypisuje wszystkie procesy, oprócz 2 procesów sesji (bash, ps w moim przypadku)
+- ps x - wypisuje wszystkie procesy aktualnego użytkownika
+- ps axu - wypisuje wszystkie procesy, wraz z ich użytkownikami
+- man ps, potem /tty
+  - ? - proces nie ma przypisanego terminala, czyli pewnie jest procesem działającym po stronie serwera
+
+- ps axu | grep bash
```

Comment:
Jeśli będziemy mieć usera o loginie `barabash` i on będzie miał uruchomiony proces `zbrodniarz` to używanie takiego filtrowania będzie błędne bo nie zwróci tylko i wyłacznie procesów bash ale zwróci dodatkowo wszystkie procesy tego usera niezależnie od tego jakie polecenia zostaną przez uruchomione. Zatem jak naprawić tę linijkę? Jakiego parametru trzeba użyc?